testing/bdd/features/timing/ebuttd_resolved_timings_on_elements.feature

```diff
@@ -74,6 +74,40 @@ Feature: Resolving timings on elements
       | 00:00:05 |          | 00:00:02    | 00:00:08  | 00:00:03         |                | 00:00:07                  | 00:00:13                | 00:00:10                       | 00:00:13                     |
       | 00:00:05 |          | 00:00:02    | 00:00:08  |                  | 00:00:06       | 00:00:07                  | 00:00:13                | 00:00:07                       | 00:00:13                     |
 
+
+  Scenario: Timings present on both p and consecutive span elements with an intervening br element
+    Given an xml file <xml_file>
+    When  it has p begin time <p_begin>
+    And   it has p end time <p_end>
+    And   it has span1 begin time <span1_begin>
+    And   it has span1 end time <span1_end>
+    And   it has br <br>
+    And   it has span2 begin time <span2_begin>
+    And   it has span2 end time <span2_end>
+    When  the document is generated
+    And   the EBU-TT-Live document is denested
+    And   the EBU-TT-Live document is valid
+    And   the EBU-TT-Live document is converted to EBU-TT-D
+    Then  EBUTTD document is valid
+    And   span1 resulted begin time is <span1_resulted_begin_time>
+    And   span1 resulted end time is <span1_resulted_end_time>
+    And   span2 resulted begin time is <span2_resulted_begin_time>
+    And   span2 resulted end time is <span2_resulted_end_time>
+    And   span2 contains one br
+    And   span3 resulted begin time is <span3_resulted_begin_time>
```

Comment:
how comes a 3rd span is created?
There isn't a 
`   And   it has span3 begin time <span2_begin>
    And   it has span3 end time <span2_end>`

Reply:
Looking at the ebutt_3_ebutt_3 file I see a `d_span_type` added 
 `        if (dataset['br_in_a_span'] is False and
           (dataset['br_begin'] is not None or dataset['br_end'] is not None))` so when a br has timing it will be wrapped in a span

Reply:
Not sure I understand your question: `<br>` cannot have its own timing, that's not allowed. It can only have timing due to its parent element having timing. `dataset['br_begin']` is set in `EBUTT3EBUTTDConverter.convert_p()`, but only if we are not going to set times on the `<p>` element. 

Is that helping?

Reply:
yeah i think i get it now, a span is created with a br element inside it but the span has the timings?

Reply:
That's correct.